src/components/App/index.tsx

```diff
@@ -0,0 +1,202 @@
+import React, { useContext } from 'react'
+import { withStyles } from '@material-ui/core/styles'
+import { SnackbarProvider } from 'notistack'
+import { connect, useSelector } from 'react-redux'
+import { useRouteMatch, useHistory } from 'react-router-dom'
+
+import AlertIcon from './assets/alert.svg'
+import CheckIcon from './assets/check.svg'
+import ErrorIcon from './assets/error.svg'
+import InfoIcon from './assets/info.svg'
+import styles from './index.module.scss'
+
+import SidebarLayout from 'src/components/SidebarLayout'
+import Header from 'src/components/SidebarLayout/Header'
+import Sidebar from 'src/components/SidebarLayout/Sidebar'
+import { SidebarContext } from 'src/components/Sidebar'
```

Comment:
Shouldn't this be part of `SidebarLayout/Sidebar`?

For me it generates a bit of confusion between `src/components/Sidebar`, `src/components/SidebarLayout`, `src/components/SidebarLayout/Sidebar`

Reply:
Agreed. I think we should rename `src/components/Sidebar` to `src/components/SafesSidebar`. 
That `SidebarContext` is not related to SidebarLayout, it's related to this:
![image](https://user-images.githubusercontent.com/4376509/90553017-1434a980-e16a-11ea-958a-c93e3890fc10.png)
That's why we should not move it IMO.

Reply:
Ok, I would be even more explicit and rename it to `src/components/SafeListSidebar`

What do you think?

Reply:
sound good. Do you want to rename it now or in another pr?

Reply:
done!

Reply:
I would move all the Sidebar related logic to `src/components/Sidebar` (including the new one)